packages/button/jest.config.js

```diff
@@ -2,4 +2,5 @@ const baseConfig = require("../../jest.config")
 
 module.exports = {
   ...baseConfig,
+  testEnvironment: "jsdom",
```

Comment:
I'm not sure there's a need for this. Did you run into any issues with `yarn test`?

Reply:
Really, this is a unnecessary change, i was tried `yarn jest --coverage` to collect coverage to guide my tests, so that way i was facing erros by wrong testEnviroment, but i can pass this env variable as param